osu.Game/OsuGame.cs

```diff
@@ -151,11 +151,11 @@ public void AddBlockingOverlay(OverlayContainer overlay)
             updateBlockingOverlayFade();
         }
 
-        public void RemoveBlockingOverlay(OverlayContainer overlay)
+        public void RemoveBlockingOverlay(OverlayContainer overlay) => Schedule(() =>
```

Comment:
Not sure about doing this here instead of locally. It raises questions about why `AddBlockingOverlay()` is not scheduled but this is, which I guess can be answered by the commit message, but it still sorta feels like this would be better placed in `OsuFocusedOverlayContainer.Dispose()` - as I'm guessing that is the only infringing usage? `UpdateState` should be safe because `UnbindAllBindables()` runs first, right?

Reply:
Not sure. That method exists only for that one class so it's a bit arbitrary where it's placed.

Reply:
Hm, fair enough. Slipped by past me that the schedule uses the implicit `this` which is an `OsuGame`.